.github/workflows/prod-cd.yml

```
@@ -132,8 +129,8 @@ jobs:
       id-token: write
       contents: read
     env:
-      IMAGE: ${{ needs.resolve.outputs.image }}
-      REGISTRY: ${{ needs.resolve.outputs.registry }}
+      TAG: ${{ needs.resolve.outputs.tag }}
+      IMAGE: ${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ needs.resolve.outputs.tag }}
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)